src/cassandra/value.rs

```diff
@@ -496,4 +496,22 @@ impl Value {
         };
         unsafe { cass_value_get_uuid(self.0, &mut output).to_result(Uuid::build(output)) }
     }
+
+    /// Get this value as a BigDecimal
+    pub fn get_decimal(&self) -> Result<BigDecimal> {
+        let mut varint = std::ptr::null();
+        let mut varint_size = 0;
+        let mut scale = 0;
+        let slice = unsafe {
+            if cass_value_get_decimal(self.0, &mut varint, &mut varint_size, &mut scale)
+                != cassandra_cpp_sys::CassError::CASS_OK
+            {
+                return Err(CASS_ERROR_LIB_INVALID_VALUE_TYPE.to_error());
```

Comment:
Can you please return the actual error, rather than always this fixed one? There's the helper `.to_result()` to help with this.

Reply:
This has been addressed.